memgpt/config.py

```diff
@@ -38,7 +38,6 @@ def set_field(config, section, field, value):
 
 @dataclass
 class MemGPTConfig:
-    config_path: str = os.path.join(MEMGPT_DIR, "config")
```

Comment:
removing config_path as a value in the config - does not add information and can create inconsistencies